packages/runtime/core/src/payout/index.ts

```diff
@@ -5,7 +5,7 @@ import { split } from "fp-ts/lib/string.js";
 import { pipe } from "fp-ts/lib/function.js";
 import { head } from "fp-ts/lib/ReadonlyNonEmptyArray.js";
 import { TxId } from "../tx/id.js";
-import { ContractId } from "../contract/id.js";
+import { ContractIdGuard } from "../contract/id.js";
 import { AssetId, Assets } from "../asset/index.js";
 
 // QUESTION: @N.H: What is the difference between PayoutId and WithdrawalId?
```

Comment:
cc @nhenin

Reply:
PayoutId is the id of the payout :slightly_smiling_face: meaning you’ll use it to withdraw the tokens from these payouts….
 WithdrawalId  are the id of the Tx  when you withdraw from this payout…